FindMe/Location/LocationManager.m

```diff
@@ -34,7 +34,7 @@ - (instancetype)init
     if (self != nil) {
         self.locationManager = [CLLocationManager new];
         self.locationManager.delegate = self;
-        self.locationManager.distanceFilter = 5;
+        self.locationManager.distanceFilter = 100;
```

Comment:
Why so high? Not hugely important, just curious.

Reply:
I used a lower five because I wanted to debug it. However in it's real world application, I thought it would be reasonable to send notification only when the location has changed by 100m